charts/service-binding-operator/README.md

```diff
@@ -19,9 +19,10 @@ The `values.yaml` file contains the following values that can be customized when
 - `image.repository`
 - `image.testRepository`
 - `keepTestResources`
+- `isOpenshift`
```

Comment:
In stead of a boolean like "is that openshift?" wouln't it be better to use a form of an enum like `kubernets.runtime` or `kubernetes.platform` with values like `vanilla`, `openshift` and `eks`?

Reply:
That way you could use the conditions like `{{ if eq .Values.kubernetes.platform openshift}}`

Reply:
I get the point but the reason i did it using isOpenshift bool is that it automatically will have just two values true or false and i would prefer installing chart with just a bool flag instead of a string flag. Let me know if it makes sense else will make the change.

Reply:
Ok, but what if you have (at some point in the future) some change that is only for EKS... you would have introduced additional boolean variable `isEks` and then allowed users to set `isOpenshift=true` and `isEks=true` at the same time which would create a potential for errors. Having it mutually excluded decreases the possibility of misuse because this way you have garanteed that the platform flag is IMHO, it doesn't matter if users need to set `isOpenshift=true` (literal `true` is a string in the shell anyway) or `platform=openshift` (`platform`, `runtime`,... whatever) , as long as they need to set it in the CLI.

Reply:
I'm inclined to agree: if we need to introduce platform-specific logic for multiple platforms, it's best if we use an enum-style type here, instead of a set of bools.